eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/BitcoinCoreClient.scala

```diff
@@ -372,7 +374,7 @@ class BitcoinCoreClient(val rpcClient: BitcoinJsonRPCClient, val onchainKeyManag
       // we ask bitcoin core to add inputs to the funding tx, and use the specified change address
       FundTransactionResponse(tx, fee, _) <- fundTransaction(partialFundingTx, FundTransactionOptions(feerate))
       lockedUtxos = tx.txIn.map(_.outPoint)
-      signedTx <- unlockIfFails(tx.txid, lockedUtxos)(sign(tx, fee, feerate))
+      signedTx <- unlockIfFails(lockedUtxos)(verifyAndSign(tx, fee, feerate))
```

Comment:
When we used to log details about that, there were a lot of such failures in the logs, mostly when utxos disappear (because they're double-spent). If we fail to unlock a utxo, we can't do anything about it apart giving up, so why bother logging anything and making the code more complex?